wiki/osu!_wiki_Contribution_Guide/es.md

```diff
@@ -0,0 +1,106 @@
+
+2\. [Inicia sesión](https://github.com/login).
+
+3\. Continúa con una [Bifurcación](#forking) del repositorio.
+
+### Bifurcación
+
+1\. Ve al [repositorio `osu-wiki`](https://github.com/ppy/osu-wiki).
+
+2\. Haz clic en `Fork`.
+
+![](img/fork.jpg)
+
+3\. Haz clic en `Fork` de nuevo para ir a tu bifurcación.
+
+4\. Si ves algo similar a la imágen significa que hiciste una bifurcación del repositorio de `ppy/osu-wiki` y estás en esa misma bifurcación.
+
+![](img/forked.jpg)
+
+5\. Continúa [Sincronizando tu bifurcación](#syncing-your-fork).
+
+### Sincronizando tu bifurcación
+
+1\. Ve al repositorio `osu-wiki`.
+
+2\. Encima de los archivos, hay un texto que te dirá si tu bifurcación está "behind", "ahead", o "even" con `ppy:master`. Si dice que tu bifurcación está "behind" o "ahead and behind" con cualquier cantidad de revisiones, significa que tu bifurcación está desactualizada.
+
+![](img/fork-even.jpg "OK \(Sin revisiones y actualizado\)")
+
+![](img/fork-ahead.jpg "OK \(Éstas son tus revisiones\)")
+
+![](img/fork-behind.jpg "Potencialmente mal \(Tu bifurcación está desactualizada\)")
+
+![](img/fork-ahead-behind.jpg "Potencialmente mal \(Tu bifurcación está desactualizada con tus revisiones\)")
+
+3\. Esto no es un gran problema; es decir, si no vas a editar un archivo que ya fue modificado, que puede ser poco probable, más revisiones tendrás.
+
+4\. Para solucionar esto, ve a [la sección ¡Mi bifurcación está desactualizada! en el artículo Problemas Comúnes](/wiki/owcg/Common_Issues/#my-branch-is-out-of-date!).
+
+5\. Una vez que hayas completado el paso 4, continúa [Editando en línea o localmente](#editing-online-or-locally).
+
+## Editando en línea o localmente
+
+Tienes dos opciones:
+
+- [Interfaz Web de GitHub](/wiki/owcg/GitHub_Web_Interface) - edita en línea; ésta es la mejor opción para la edición de un solo archivo
+- [GitHub para Escritorio](/wiki/owcg/GitHub_Desktop) - edita localmente; ésta es la mejor opción para la edición de uno o más archivos (include subir, eliminar y mover imágenes o archivos)
+
+*Nota: Puedes eligir cualquiera de las dos opciones listadas para editar.* Para mantener esta guía corta, estos artículos no discutirán el uso de otras herramientas en lo absoluto. Hay otras aplicaciones de terceros que pueden hacer más o menos de lo que la Interfaz Web de GitHub y GitHub para Escritorio pueden hacer.
+
+**Cuando hayas completado la edición de tu archivo, ya sea con la *Interfaz Web de GitHub* o *GitHub para Escritorio*, puedes continuar con la siguiente sección.**
+
+## Finalizando
+
+### Abrir un "pull request"
+
+1\. Ve al [repositorio `ppy/osu-wiki`](https://github.com/ppy/osu-wiki).
+
+2\. Si fuiste lo suficientemente rápido, es posible que veas este cartel amarillo
+
+![](img/github-recent.jpg)
+
+3\. Si lo ves, haz clic en el botón `Compare & pull request` (saltar hasta el paso 8). Si no, haz clic en  el botón `New pull request`(continúa con el paso 4).
```

Comment:
Between en and el, there are two spaces. Only shows in code. Shows as one space in preview.